contrib/grpcplugins/action/plugin-venom/main.go

```diff
@@ -153,11 +154,17 @@ func (actPlugin *venomActionPlugin) Run(ctx context.Context, q *actionplugin.Act
 		}
 	}
 
+    stopOnFailure, err := strconv.ParseBool(stopOnFailureStr)
```

Comment:
you have to check if stopOnFailureStr is not empty. 

If empty, stopOnFailureStr should be set to false with no error returned (without that, all existing users using plugin-venom without the stopOnFailureStr set will have an error as: strconv.ParseBool: parsing "": invalid syntax)

Reply:
+ I'm not sure that you have gofmt your code